Gemfile.lock

```diff
@@ -162,7 +162,7 @@ GEM
     multi_json (1.13.1)
     multi_xml (0.5.5)
     multipart-post (2.0.0)
-    nokogiri (1.8.2)
+    nokogiri (1.8.4)
```

Comment:
Is this really in this PR? Or is it a dup of our update last week?

Reply:
It made it to BitBucket but not to GitHub given the breaking builds in Bamboo. This will be the first PR merged into e2e-services on GitHub since the update.